Gemfile.lock

```diff
@@ -499,38 +503,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
+    parallel_tests (2.32.0)
```

Comment:
### [parallel_tests](https://github.com/grosser/parallel_tests)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.21.1 → 2.32.0

[_~~change-log~~, [source-code](https://github.com/grosser/parallel_tests/tree/v2.32.0), [gem-diff](https://my.diffend.io/gems/parallel_tests/2.21.1/2.32.0)_]





<details>
<summary>Commits</summary>

A change of **169** commits. See the full changes on [the compare page](https://github.com/grosser/parallel_tests/compare/v2.21.1...v2.32.0).

These are the first 10 commits:
- (7ac4890) [load library before using methods from it](https://github.com/grosser/parallel_tests/commit/7ac4890d6c1a34b9a4d361288a8b52c14a8d669c)
- (9c8bdfa) [thx for the fix](https://github.com/grosser/parallel_tests/commit/9c8bdfad9406fdd22b3d793a216527bc4348a315)
- (dbbf710) [[CI] Test against new Ruby 2.x versions](https://github.com/grosser/parallel_tests/commit/dbbf710666e1b905e5968dbe7b00805300c54821)
- (9453746) [[CI] Test against new Ruby 2.4 version](https://github.com/grosser/parallel_tests/commit/9453746ae35f36c52cbbb4a4adf0335bb8c1fa4c)
- (38bc732) [Fix fixtures for json gem on Ruby >= 2.4](https://github.com/grosser/parallel_tests/commit/38bc73295361a832484b397cfd8cb01d680d7daa)
- (0142e1d) [Fix Rails 4.2 fixture for Ruby >= 2.4](https://github.com/grosser/parallel_tests/commit/0142e1d8b6dbde76df38959e14da1362185aa53e)
- (984570d) [Fix Rails 3.2 fixture for Ruby >= 2.4](https://github.com/grosser/parallel_tests/commit/984570d7294a4fa4b234392bd086960c8f23e92d)
- (a123087) [[CI] Hide verbose Ruby 2.5 warnings](https://github.com/grosser/parallel_tests/commit/a123087e80aeaaa9d8df4421bd8114301d1a9578)
- (fba5c5e) [[Test] Fix tests with Rails 4.2 and nokogiri](https://github.com/grosser/parallel_tests/commit/fba5c5e935045d583e60415b71b403c9f004887e)
- (00a9943) [Merge pull request #621 from nicolasleger/ci-new-ruby-2](https://github.com/grosser/parallel_tests/commit/00a99433f1a99f9b2dcb97f229e175b950cdd54d)


</details>